lib/player_manager.rb

```diff
@@ -1,23 +1,31 @@
-require 'game_state'
+require 'ai_basic'
```

Comment:
You're right they are not used and I could and should removed those two requires.